extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,139 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
+.i-amp-story-quiz-container {
```

Comment:
Can you rename all `i-amp-*` into `i-amphtml-*`? That's the prefix used by AMP extensions